idaes/models_extra/temperature_swing_adsorption/fixed_bed_tsa0d.py

```diff
@@ -0,0 +1,4563 @@
+        if (
+            self.config.transformation_method == "dae.finite_difference"
+            and self.config.transformation_scheme is None
+        ):
+            self.config.transformation_scheme = "BACKWARD"
+        elif (
+            self.config.transformation_method == "dae.collocation"
+            and self.config.transformation_scheme is None
+        ):
+            self.config.transformation_scheme = "LAGRANGE-RADAU"
+        elif (
+            self.config.transformation_method == "dae.finite_difference"
+            and self.config.transformation_scheme != "BACKWARD"
+            and self.config.transformation_scheme != "FORWARD"
+        ):
+            raise ConfigurationError(
+                "{} invalid value for "
+                "transformation_scheme argument. "
+                "Must be "
+                "BACKWARD"
+                " or "
+                "FORWARD"
+                " "
+                "if transformation_method is"
+                " "
+                "dae.finite_difference"
+                ".".format(self.name)
+            )
+        elif (
+            self.config.transformation_method == "dae.collocation"
+            and self.config.transformation_scheme != "LAGRANGE-RADAU"
+        ):
+            raise ConfigurationError(
+                "{} invalid value for "
+                "transformation_scheme argument."
+                "Must be "
+                "LAGRANGE-RADAU"
+                " if "
+                "transformation_method is"
+                " "
+                "dae.collocation"
+                ".".format(self.name)
+            )
+
+        # consistency check for properties
+        if self.config.compressor and self.config.compressor_properties == useDefault:
+            raise ConfigurationError(
+                "Compressor flag was set to true "
+                "but no property package was provided"
+            )
+
+        if (
+            self.config.steam_calculation == "rigorous"
+            and self.config.steam_properties == useDefault
+        ):
+            raise ConfigurationError(
+                "Steam calculations are set to rigorous "
+                "but no property package was provided"
+            )
+
+        # add required parameters
+        self._add_general_parameters()
+
+        # add adsorbent parameters
+        if self.config.adsorbent == "Zeolite-13X":
+            self._add_parameters_zeolite_13x()
+        elif self.config.adsorbent == "mmen-Mg-MOF-74":
+            self._add_parameters_mmen_Mg_MOF_74()
+        elif self.config.adsorbent == "polystyrene-amine":
+            self._add_parameters_polystyrene_amine()
+
+        # add design and operating variables
+        self.flow_mol_in_total = Var(
+            initialize=0.014,
+            domain=PositiveReals,
+            units=units.mol / units.s,
+            doc="Total mole flow rate at inlet [mol/s]",
```

Comment:
I would suggest not including the units in the doc string as this just duplicates the units attribute (and opens up the possibility of them conflicting).

Reply:
Removed units from doc strings